CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Head
+
+- Removed: dependency on Stylelint's internal module `lib/utils/getOsEol`.
```

Comment:
[ask] Should we release this change as a *breaking change*?

Reply:
Is it breaking for the user? It seems closer to a refactor. How about patch?

> - Fixed: tightly-coupled dependency on Stylelint's internal module `lib/utils/getOsEol`.

Reply:
```suggestion
- Fixed: tightly-coupled dependency on Stylelint's internal module `lib/utils/getOsEol`.
```